src/Package/Impl/DataInspect/VariableView.xaml

```diff
@@ -89,16 +90,18 @@
             </DataTemplate>
 
             <DataTemplate x:Key="ValueCellTemplate">
-                <Grid>
+                <Grid Height="{Binding Path=RowHeight, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type DataGrid}}, Converter={x:Static rwpf:Converters.Scale155}}"
```

Comment:
Can you reference it using `ElementName` rather than `RelativeSource Mode=FindAncestor`?